DOCKER/Dockerfile

```diff
@@ -22,17 +22,32 @@ RUN apk update && \
 # Run the container with tmuser by default. (UID=100, GID=1000)
 USER tmuser
 
-# Expose the data directory as a volume since there's mutable state in there
-VOLUME [ $TMHOME ]
-
 WORKDIR $TMHOME
 
 # p2p and rpc port
-EXPOSE 26656 26657
+EXPOSE 26656 26657 26660
 
 ENTRYPOINT ["/usr/bin/tendermint"]
-CMD ["node", "--moniker=`hostname`"]
+CMD ["node"]
 STOPSIGNAL SIGTERM
 
 ARG BINARY=tendermint
 COPY $BINARY /usr/bin/tendermint
+
+# Create default configuration for docker run.
+RUN /usr/bin/tendermint init && \
+    sed -i \
+      -e 's/^proxy_app\s*=.*/proxy_app = "kvstore"/' \
+      -e 's/^moniker\s*=.*/moniker = "dockernode"/' \
+      -e 's/^addr_book_strict\s*=.*/addr_book_strict = false/' \
+      -e 's/^timeout_commit\s*=.*/timeout_commit = "500ms"/' \
```

Comment:
why default got changed here?

Reply:
Good point. The defaults are good enough, but they are optimized for multi-node networks. In this small single-node example, it makes sense to make the network somewhat faster. I was contemplating turning on `skip_timeout_commit` since there's no one else to wait for, but it made the network so fast (tens of blocks per second) that it would've been harder for developers to see what's happening. Changing the `timeout_commit` makes it look cooler during development. :)

In a current use-case, we have a test in the tendermint-rs repository that checks if at least 10 blocks were created. Essentially, the test needs to sleep until 10 blocks were created. Normally, this would take 10 seconds, but with the modification, it's only 5.